src/styles.css

```diff
@@ -135,46 +141,53 @@ a:hover {
 .recipe-card-name {
   padding-bottom: 10px;
   margin:10px auto;
-  width:90%;
-  text-align: center;
+  width:100%;
+  height:60px;
+  text-align: left;
   font-weight: bold;
-  border-bottom: 1px solid rgb(43, 43, 43)
+  border-bottom: 1px dotted rgb(179, 178, 178)
 }
 
```

Comment:
Recipe mini cards look much better with even spacing! The names might need a little left/right padding since they're sitting right on the border of the card. Other than that, looks great!